packages/ui/src/components/Badge/Badge.stories.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { Meta, Story } from '@storybook/react';
+
+import { HStack } from '../Stack';
+import { Badge, BadgeProps } from './Badge';
+
+const meta: Meta = {
+  title: 'Badge',
+  component: Badge,
+};
+
+export default meta;
+
+const Template: Story<BadgeProps> = (args) => <Badge {...args} />;
+
+export const Playground = Template.bind({});
+
+Playground.args = {
+  children: 'Badge text',
+  size: 'large',
+  color: 'brand',
+};
+
+export const Sizes = () => (
+  <HStack>
+    <Badge size="small" color="red">
```

Comment:
A small thing, but if we want to show how the component displays with different `size` prop, we shouldn't pass other props like `color`. I think you can just leave the default value here.